cmd/cachecmd/main.go

```diff
@@ -327,11 +341,16 @@ func cacheDir() string {
 
 // REF: https://specifications.freedesktop.org/basedir-spec/basedir-spec-0.6.html
 func xdgCacheHome() string {
-	path := os.Getenv("XDG_CACHE_HOME")
-	if path == "" {
-		path = filepath.Join(os.Getenv("HOME"), ".cache")
+	home := ""
+	if runtime.GOOS == "windows" {
+		home = os.Getenv("USERPROFILE")
+	} else {
+		home = os.Getenv("XDG_CACHE_HOME")
```

Comment:
XDG_CACHE_HOME is including "/.cache", so we should not append ".cache".  https://specifications.freedesktop.org/basedir-spec/basedir-spec-0.6.html